features/step_definitions/group_selection_steps.rb

```diff
@@ -419,6 +436,31 @@
   end
 end
 
+And(/.* should see eligibility failed error on dependent with resident role/) do
+  expect(find('.ineligible_ivl_row .ivl_errors')).to have_content("eligibility failed on active_individual_role")
+  expect(find(IvlChooseCoverage.dependent_1_checkbox).checked?).to be_falsey
+end
+
+Then(/.* should see all family members eligible$/) do
+  expect(find(IvlChooseCoverage.primary_checkbox).checked?).to be_truthy
+  expect(find(IvlChooseCoverage.dependent_1_checkbox).checked?).to be_truthy
+end
+
+And(/.* should see warning dialog on CoverAll selection/) do
+  expect(find("#WarningOnCoverAllSelection").visible?).to be_truthy
+
+  within('#WarningOnCoverAllSelection .modal-dialog') do
+    expect(page).to have_selector('.modal-title', text: 'Some people are eligible for DC Health Link')
+    expect(page).to have_selector('.modal-cancel-button', text: /close/i)
+  end
+end
+
+Then(/.* clicked close on CoverAll selection warning dialog/) do
+  within('#WarningOnCoverAllSelection .modal-dialog') do
+    find('.modal-cancel-button', text: /close/i).click
```

Comment:
We should use an available translation via the l10n helper method here so that we don't create a dependency between the hard coded English string and the Cucumber.
```
l10n("close")
```

Reply:
done